cisco_firepower_connector.py

```diff
@@ -333,15 +341,16 @@ def _gen_network_dict(self):
         from the ip parameter value.
         """
         ip_and_mask = self.destination_network.split("/")
-        if len(ip_and_mask) == 1 or (self.ip_version == 4 and int(ip_and_mask[1]) == 32) or \
-                (self.ip_version == 6 and int(ip_and_mask[1]) == 128):
+        if (
+            len(ip_and_mask) == 1  # noqa
+            or (self.ip_version == 4 and int(ip_and_mask[1]) == 32)  # noqa
+            or (self.ip_version == 6 and int(ip_and_mask[1]) == 128)  # noqa
+        ):
             self.debug_print("IP is type Host")
-            self.destination_dict = {"type": "Host",
-                                     "value": "{0}".format(self.destination_network)}
+            self.destination_dict = {"type": "Host", "value": "{0}".format(self.destination_network)}
         elif len(ip_and_mask) == 2:
             self.debug_print("IP is type Network")
-            self.destination_dict = {"type": "Network",
-                                     "value": "{0}".format(self.destination_network)}
+            self.destination_dict = {"type": "Network", "value": "{0}".format(self.destination_network)}
         self.debug_print("Network Dictionary: {0}".format(self.destination_dict))
```

Comment:
```suggestion
        self.debug_print("Network Dictionary: {str(self.destination_dict)}")
```

to be consistent this should be updated as well